src/index.html

```diff
@@ -7,6 +7,435 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="moyo" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="moyo"
+          class="header__logo-main"
+        >
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="header__menu">
+          <li class="header__menu-item">
+            <a href="Apple" class="header__menu-link is-active">Apple</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="Samsung" class="header__menu-link">Samsung</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="Smartphones" class="header__menu-link">Smartphones</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a
+              href="Laptops & Computers"
+              class="header__menu-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="Gadgets" class="header__menu-link">Gadgets</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="Tablets" class="header__menu-link">Tablets</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="Photo" class="header__menu-link">Photo</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="Video" class="header__menu-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards">
+      <div class="cards__item" data-qa="cards">
+        <img
+          src="./images/imac.jpeg"
+          alt="product-image"
+          class="cards__img"
+        >
+
+        <h2 class="cards__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="cards__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="cards__rang">
+          <div class="stars cards__stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="cards__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="cards__expense">
+          <p class="cards__expense-price">
+            Price:
+          </p>
+
+          <p class="cards__expense-cost">
+            $2,199
+          </p>
+        </div>
+
+        <button
+          type="button"
+          class="cards__button"
+          data-qa="cards-hover"
+        >
+          Buy
+        </button>
+      </div>
+      <div class="cards__item">
```

Comment:
Add a blank line between the cards
```suggestion
      </div>
      
      <div class="cards__item">
```